backend/curation/api/v1/curation/collections/common.py

```diff
@@ -222,6 +222,7 @@ def reshape_dataset_for_curation_api(
 
     ds["dataset_id"] = dataset_version.dataset_id.id
     ds["dataset_version_id"] = dataset_version.version_id.id
+    ds["primary_cell_count"] = dataset_version.metadata.primary_cell_count
```

Comment:
remove, metadata fields are already included as part of this block: https://github.com/chanzuckerberg/single-cell-data-portal/pull/5705/files#diff-26c746d9f1e6e25a74bd8918b8773e6352c44219da15aeb38662790fbc721608R210-R221

Reply:
Add 'primary_cell_count' to https://github.com/chanzuckerberg/single-cell-data-portal/blob/631a0dfde8672d24272c9c09b169a04d11e9002c/backend/curation/api/v1/curation/collections/common.py#L299 instead